Add ModflaredApi

Skidamek opened the pull request:
#398

FlamiDev commented:
Hey, just out of interest, how far along is this pr? It would be super useful for our private server, and I'm really happy you're already working on a solution. Is there anything I could help with? I'm a pretty experienced dev and while I don't have much experience modding I could pick it up pretty easily. Either way, looking forward to when this gets merged!

Skidamek commented:
> Hey, just out of interest, how far along is this pr? It would be super useful for our private server, and I'm really happy you're already working on a solution. Is there anything I could help with? I'm a pretty experienced dev and while I don't have much experience modding I could pick it up pretty easily. Either way, looking forward to when this gets merged!

Yeah, the modflared api would need to be developed, implemented properly into automodpack and fixed.
https://github.com/HttpRafa/modflared/pull/75
If you know how to make good enough, working api as specified in the pr that'd help a lot, feel free to play with it.
